scripts/flow/flowListener.js

```diff
@@ -51,17 +70,47 @@ const _bindFlowHook = ({ flowName, fallbackActionIdentifier = null }) => {
     });
 };
 
+/* -------------------------------------------- */
+
+const fallbackActionIdentifier_BasicAttackFlow = flow => {
+    return flow.state.data.attack_type === "Melee" ? "lwfx_default_melee" : "lwfx_default_ranged";
+};
+
+const fallbackActionIdentifier_StructureFlow = flow => {
+    switch (flow.state.data.title) {
```

Comment:
Admittedly "title" is not the best switch, as e.g. a translation could change it in future. Currently in the Lancer system these titles are hard-coded: https://github.com/Eranziel/foundryvtt-lancer/blob/master/src/module/flows/structure.ts#L103
So this will work for now. In future, if the strings are translated, it will be trivial to switch on the translated title instead.

This is still much easier than re-implementing the structure damage/overheating tables ourselves, and it works!